src/humidity.jl

```diff
@@ -0,0 +1,68 @@
+
+"""
+Humidity struct containing relevant parameters and arrays
+"""
+struct compute_relative_and_saturation_humidity{NF<:AbstractFloat}     
+
+#Set of constants used in the saturation humidity calculations
+#Where do these come from? What do they represent?
+
+e0 ::Float64= 6.108e-3
+c1 ::Float64= 17.269
+c2 ::Float64= 21.875
+t0 ::Float64= 273.16 #Paxton/Chantry have a zero_c term here? 
```

Comment:
They might have done so to avoid the Kelvin vs Celsius offset for precision reasons?